regulations/static/regulations/js/source/views/main/main-view.js

```diff
@@ -100,9 +101,11 @@ var MainView = Backbone.View.extend({
             this.childView = new this.viewmap[this.contentType](childViewOptions);
         }
 
-        this.commentViews = this.$el.find('.comment-wrapper').map(function(idx, elm) {
-            return new CommentView({el: elm});
-        });
+        if (this.commentTypes.indexOf(this.contentType) !== -1) {
+            this.commentViews = this.$el.find('.comment-wrapper').map(function(idx, elm) {
+                return new CommentView({el: elm, hide: true});
+            });
+        }
```

Comment:
Is there a way to get this working _with_ the existing JS rather than making an exception, here?